Integrations/integration-SymantecEndpointProtection_V2.yml

```diff
@@ -0,0 +1,1136 @@
+            }
+        })
+
+    def list_policies_command(token):
+        policies_list = do_get(token, False, 'sepm/api/v1/policies/summary').get('content')
+
+        fixed_policie_list = sanitize_policies_list(policies_list)
+        md_list = sanitize_policies_list_for_md(fixed_policie_list)
+        md = tableToMarkdown('List of existing policies', md_list, ['Policy Name', 'Type', 'ID', 'Enabled', 'Assigned', 'Discription'])
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': policies_list,
+            'HumanReadable': md,
+            'IgnoreAutoExtract': True,
+            'EntryContext': {
+                'SEPM.PoliciesList': createContext(fixed_policie_list, removeNull=True)
+            }
+        })
+
+
+    def assign_policie_command(token):
+        group_id = demisto.getArg('groupID')
+        locatoion_id = demisto.getArg('locationID')
+        policy_type = demisto.getArg('policyType').lower()
+        policy_id = demisto.getArg('policyID')
+        resp = do_put(token, 'sepm/api/v1/groups/{0}/locations/{1}/policies/{2}'.format(group_id, locatoion_id, policy_type), {'id': policy_id})
+        md = '### Policy: {0}, of type: {1}, was assigned to location: {2}, in group: {3}'.format(policy_id, policy_type, locatoion_id, group_id)
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': '',
+            'HumanReadable': md,
+            'EntryContext': {}
+        })
+
+
+    def list_locations_command(token):
+        group_id = demisto.getArg('groupID')
+        url = 'sepm/api/v1/groups/{}/locations'.format(group_id)
+        url_resp = do_get(token, False, url)
+        location_ids = map(lambda location_string: {'ID': location_string.split('/')[-1]}, url_resp)
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': url_resp,
+            'HumanReadable': tableToMarkdown('Locations', map(lambda location: {'Location ID': location.get('ID')}, location_ids)),
+            'IgnoreAutoExtract': True,
+            'EntryContext': {
+                'SEPM.Locations': location_ids
+            }
+        })
+
+
+    def endpoint_quarantine_command(token):
+        endpoint = demisto.getArg('endpoint')
+        action = demisto.getArg('actionType')
+        action_type = 'Quarantine' if action == 'Add' else 'Undo'
+
+        if not endpoint:
+            return_error('Please provide the IP address or the hostname of endpoint.')
+        elif validate_ip(endpoint):
+            computer_id = get_computer_id(token, endpoint, None)
+        else:
+            computer_id = get_computer_id(token, None, endpoint)
+        command_id = quarantine(token, computer_id, action_type)
```

Comment:
i think lines 979-987 should be in business logic functions

Reply:
- [ ] Status